test/services/assets/handlers/python.spec.js

```diff
@@ -427,65 +427,67 @@ describe('services', () => {
         expect(libraries.length).toEqual(1);
         expect(libraries[0]).toMatchObject({
           id: 'open - "test.txt"',
           type: 'data',
-          path: '"test.txt"'
+          path: '"test.txt"',
         });
         libraries = new PythonHandler().getOutputs('test.uri', 'f = open("test.dat", "w")');
         expect(libraries.length).toEqual(1);
         expect(libraries[0]).toMatchObject({
           id: 'open - "test.dat"',
           type: 'data',
-          path: '"test.dat"'
+          path: '"test.dat"',
         });
         libraries = new PythonHandler().getOutputs(
           'test.uri',
-          "f = open('test.tmp', 'w', encoding='utf-8')"
+          "f = open('test.tmp', 'w', encoding='utf-8')",
         );
         expect(libraries.length).toEqual(1);
         expect(libraries[0]).toMatchObject({
           id: "open - 'test.tmp'",
           type: 'data',
-          path: "'test.tmp'"
+          path: "'test.tmp'",
         });
         libraries = new PythonHandler().getOutputs(
           'test.uri',
-          "f = open ( 'test.tmp', 'w'  ,   encoding='utf-8' ) "
+          "f = open ( 'test.tmp', 'w'  ,   encoding='utf-8' ) ",
         );
         expect(libraries.length).toEqual(1);
         expect(libraries[0]).toMatchObject({
           id: "open - 'test.tmp'",
           type: 'data',
-          path: "'test.tmp'"
+          path: "'test.tmp'",
         });
       });
       it('should handle multiple IO open statements in a single string', () => {
         const libraries = new PythonHandler().getOutputs(
           'test.uri',
-          'f = open("test.dat", "w")\r\n\r\nprint("Hello world")\r\n\r\n\t  f = open("test.txt", "wb", buffering=0)  \r\nf = open ( "test.dat" ) '
+          'f = open("test.dat", "w")\r\n\r\nprint("Hello world")\r\n\r\n\t  f = open("test.txt", "wb", buffering=0)  \r\nf = open ( "test.dat" ) ',
         );
         expect(libraries.length).toEqual(2);
       });
       it('should ignore save locations for IO output when command case mismatches', () => {
         const libraries = new PythonHandler().getOutputs(
           'test.uri',
-          "OPEN('C:\\dev\\test.png', 'w')"
+          "OPEN('C:\\dev\\test.png', 'w')",
         );
         expect(libraries.length).toEqual(0);
       });
       it('should ignore commented out IO output lines', () => {
         expect(
-          new PythonHandler().getOutputs('test.uri', '# f = open("test.dat", "w")').length
+          new PythonHandler().getOutputs('test.uri', '# f = open("test.dat", "w")').length,
         ).toEqual(0);
         expect(
-          new PythonHandler().getOutputs('test.uri', "   #df.to_csv('test.csv')").length
+          new PythonHandler().getOutputs('test.uri', "   #df.to_csv('test.csv')").length,
         ).toEqual(0);
         // eslint-disable-next-line prettier/prettier
-        expect(new PythonHandler().getOutputs('test.uri', '   #    f = open("test.dat", "w")').length).toEqual(0);
+        expect(
```

Comment:
I'm okay to accept the linter suggestion here, but we should remove the eslint-disable line on 487